source/_integrations/synology.markdown

```diff
@@ -1,38 +1,37 @@
 ---
 title: Synology
-description: Instructions on how to integrate Synology Surveillance Station cameras within Home Assistant.
+description: Instructions on how to integrate Synology Surveillance Station within Home Assistant.
 logo: synology.png
 ha_category:
   - Camera
+  - Switch
 ha_release: 0.31
 ha_iot_class: Local Polling
 ---
 
-The `synology` camera platform allows you to watch the live streams of your [Synology](https://www.synology.com/) Surveillance Station based IP cameras in Home Assistant.
+The `synology` platform allows you to integrate your [Synology](https://www.synology.com/) Surveillance Station with Home Assistant.
 
-<div class='note'>
-
-Synology has disabled the livestreaming API and the integration is currently broken if you are using Surveillance Station version 8.2.3-5828.
-There is an unsupported preview fix available. (8.2.3-5829) - Instructions can be found [here](https://www.vcloudinfo.com/2019/04/how-to-manually-upgrade-your-synology-surveillance-system-firmware.html) for updating manually.
+There is currently support for the following device:
 
-</div>
+- [Camera](/integrations/synology/#camera)
+- [Switch](/integrations/synology/#switch)
 
 ## Configuration
 
-To enable your Surveillance Station cameras in your installation, add the following to your `configuration.yaml` file:
+To enable your Surveillance Station in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
-# Minimum configuration.yaml entry
-camera:
-  - platform: synology
-    url: IP_ADDRESS_OF_SYNOLOGY_NAS
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
+synology:
+  name: "Synology NAS"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Please remove `name` & `verify_ssl`

Reply:
Hi @frenck. Thank you for the review! I have removed the optional config parameters.